data/tutorials/gs_00_up_and_running.md

```diff
@@ -27,90 +27,119 @@ for the instruction.
+$ apt install opam
 
 # Archlinux
-pacman -S opam
+$ pacman -S opam
 
 # Debian (stable, testing and unstable)
-apt-get install opam
+$ apt-get install opam
 ```
 
-**Binary distribution**
+**Binary Distribution**
 
 ```shell
-bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
+$ bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
 ```
 
-### Initialising `opam`
+### Initialising opam
+
+It's essential to both initialise opam and install the OCaml compiler because OCaml is a compiled language, so please don't skip this section. 
 
 This step is done automatically by the alpha-version platform installer, [see below](#up-and-running-with-the-platform-installer).
 
-If you have installed the binary distribution of `opam` through the install script, this step should already be done. If you have installed it through your system package manager, you must initialise `opam` by running the following command:
+If you have installed the binary distribution of opam through the install script, this step should already be done. If you have installed it through your system package manager, you must initialise opam by running the following command:
 
-```
-opam init          # Can take some time
-eval $(opam env)
+```shell
+$ opam init          # Can take some time
+$ eval $(opam env)
 ```
 
-The first command (`opam init`) creates a first switch, usually called `default`, although this is just a convention. If you have installed OCaml through your system package manager, the first switch will be set up to use this compiler (it is called a "system switch"). Otherwise, it will build one from source, usually taking the most recent version of OCaml.
+The first command (`opam init`) creates a first switch, usually called `default`, although this is just a convention. (`switch` is used to have several installations on disk, like packages, compiler version, etc.) If you have installed OCaml through your system package manager, the first switch will be set up to use this compiler (it is called a "system switch"). Otherwise, it will build one from source, usually taking the most recent version of OCaml.
 
 The second command (`eval $(opam env)`) modifies a few environments variables to make the shell aware of the switch you are using. For instance, it will add what is needed to the `PATH` variable so that typing `ocaml` in the shell runs the OCaml binary of the current switch.
 
-In case you are not satisfied with the OCaml version of your system switch, you can write the following commands to create a new switch with a recent version of OCaml:
+**Please note:** After the `opam init` command, you might get a result asking if you'd like to update your `zsh` configuration. If you get that message, type in `N`, the default, then type `y` to install the hook, which will run `eval $(opam env)`. As you get more well-versed in OCaml, you can change these settings by rerunning `opam init`.
```

Comment:
I included an explanation on this in my previous suggestion (I think it is better grouping info on `opam init` and only after info on `eval $(opam env)`)

Reply:
I think this sentence should be change for different reasons:
- Not everyone has `zsh`, the prompt will be different depending on the shell (`zsh`, `bash`, ...) of the user. So we should not specify it!
- It is a bit confusing that to answer "yes" to a single question, the user has to type twice a letter, and even more that it starts with N for No when we want to answer Yes! I think that's a mistake: only `y` has to be typed.
- It is better to explain what it does than saying: type this, then type that...

So, I suggested this modificatio in https://github.com/ocaml/ocaml.org/pull/548#discussion_r951183090 :

```suggestion
At the end of the `opam init`, you are asked if you want to add a hook to your shell to best integrate with your system. Indeed, in order for the shell to be aware of the tools available in the current opam switch, a few environments variables need to be modified. For instance, the `PATH` variable has to be expanded so that typing `ocaml` in the shell runs the OCaml binary _of the current switch_. Answering `y` will provide a better user experience.
```